src/main/resources/META-INF/plugin.xml

```diff
@@ -1,13 +1,13 @@
 <idea-plugin>
     <id>detekt</id>
-    <name>detekt</name>
+    <name>Detekt</name>
```

Comment:
I am unsure if this is backwards compatible. I am going to publish it and will fix it myself in case this does not work.